agent/app/agent_unix.go

```diff
@@ -171,12 +170,12 @@ func (agent *ecsAgent) verifyCNIPluginsCapabilities() error {
 // notifications from the monitor
 func (agent *ecsAgent) startUdevWatcher(state dockerstate.TaskEngineState, stateChangeEvents chan<- statechange.Event) error {
 	seelog.Debug("Setting up ENI Watcher")
-	udevMonitor, err := udevwrapper.New()
+	// Create Watcher
+	eniWatcher, err := watcher.New(agent.ctx, agent.mac, state, stateChangeEvents)
 	if err != nil {
-		return errors.Wrapf(err, "unable to create udev monitor")
+		return errors.Wrapf(err, "unable to start udevWatcher")
```

Comment:
since udev is now an implementation detail of the eni watcher, this message can just be "unable to create ENI watcher"